static/src/javascripts/projects/commercial/modules/header-bidding/prebid/bid-config.spec.js

```diff
@@ -524,3 +525,69 @@ describe('triplelift adapter', () => {
         });
     });
 });
+
+describe('getXaxisPlacementId', () => {
+    beforeEach(() => {
+        resetConfig();
+        getBreakpointKey.mockReturnValue('D');
+
+        containsMpuOrDmpu.mockReturnValueOnce(true)
+            .mockReturnValueOnce(true)
+            .mockReturnValue(false);
+        containsLeaderboardOrBillboard.mockReturnValueOnce(true)
+            .mockReturnValueOnce(true)
+            .mockReturnValue(false);
+    });
+
+    afterEach(() => {
+        jest.resetAllMocks();
+    });
+
+    const generateTestIds = () => {
+        const prebidSizes = [
+            [[300, 250]],
+            [[300, 600]],
+            [[970, 250]],
+            [[728, 90]],
+            [[1, 2]],
+        ];
+        return prebidSizes.map(getXaxisPlacementId);
```

Comment:
There’s been some chat around this recently, and the danger around the fact that map has arity 3: https://jakearchibald.com/2021/function-callback-risks/